nixos/tests/networking.nix

```diff
@@ -101,9 +101,6 @@ let
             { address = "192.168.1.3"; prefixLength = 32; }
             { address = "192.168.1.10"; prefixLength = 32; }
           ];
-          interfaces.eth2.ipv4.addresses = mkOverride 0 [
-            { address = "192.168.2.2"; prefixLength = 24; }
-          ];
```

Comment:
is this an intentional change? removing this breaks the test.

Reply:
Oops. Sorry that must have been while I was figuring out how the tests worked!  Should be reinstated now.